pytorch_binding/setup.py

```diff
@@ -8,10 +8,10 @@
 
 
 extra_compile_args = ['-fPIC']
-if LooseVersion(torch.__version__) >= LooseVersion("1.5.0"):
```

Comment:
What's the reason to use different c++ version? Why we can not use c++17 for any PyTorch version?

Reply:
PyTorch master has switched to C++17, need to update C++ version of this repo, otherwise, it will build fail. I have tested  pytorch 1.13.0, C++17 works well. Suppose we can use C++17 for any PyTorch version.